core/scss/components/input/input.scss

```diff
@@ -321,8 +321,10 @@
     &.#{$input__class}--filled {
       @include input-color(
         $background-color: $input__background--filled,
+        $background-color-css-var: --md-background-tertiary,
         $background-color-disabled: $input__background--filled,
-        $background-color-hover: $input__background--filled--hover
+        $background-color-hover: $input__background--filled--hover,
+        $background-color-hover-css-var: --md-background-quaternary,
```

Comment:
Do we want to include `$background-color-disabled-css-var`? Not sure if this is a part of the mixin.

Reply:
It's in the mixin as well.